packages/config/src/mk/build.mk

```diff
@@ -1,15 +1,17 @@
 .PHONY: .build
 .build: $(if $(NOPRUNE),,prune) install
-	@npx vite \
-		-c ./vite.config.production.ts \
-		build
+	@NODE_OPTIONS=--experimental-strip-types \
+		npx vite \
```

Comment:
WOuld be great in the future at some point if we centralize some sort of `NODE_RUNNER` or `JS_RUNNER`/`CLI_RUNNER` or something, so that we specify this in one place. Totally a future things though.

Reply:
Actually, does it matter that we don't always need these options when using `npx`, anyway something to think about.

Reply:
@schogges was saying, maybe we think of it like a `VITE_RUNNER`, which sounds like what we are saying here, although ideally without `VITE_` ie agnostic to the tool, anyway food for thought.